recipes/healpix-geo/recipe.yaml

```diff
@@ -0,0 +1,62 @@
+context:
+  name: healpix-geo
+  version: "0.0.6"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/eopf-dggs/healpix-geo/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 88ee6fe9c018387580a0dff03c04e3dae9b65ac3cf002418f5fc659a2db1bde4
+
+build:
```

Comment:
I would add:

```
skip:
   - match(python, "<3.10")
```

under `build`. While conda-forge is about to stop building packages for 3.9, we are right at the transition period and it is probably good to add this as a precaution.